src/fidesops/ops/service/connectors/saas/connector_registry_service.py

```diff
@@ -123,7 +128,7 @@ def create_dataset_config_from_template(
         "fides_key": template_values.instance_key,
         "dataset": dataset_from_template,
     }
-    dataset_config = DatasetConfig.create(db, data=data)
+    dataset_config = DatasetConfig.create_or_update(db, data=data)
```

Comment:
This seems fine given that the endpoint that creates the resources from a template initially has an early check to make sure that the DatasetConfig fides_key doesn't already exist.

Reply:
right, and i wavered between just subbing this in or having the function take another param to indicate whether a create or an update was desired.

i can see how there's a chance this functionality could be missed by someone else calling this function. i'll update the docstring of the function to make it clear that it creates or updates.

let me know if you think we should approach it differently

Reply:
The function name could be changed too, I think that would also help!